internal/integration/postgres_test.go

```diff
@@ -1119,11 +1119,14 @@ create table atlas_types_sanity
 }
 
 func (t *pgTest) url() string {
-	return t.dsn()
+	return t.dsn("")
 }
 
-func (t *pgTest) dsn() string {
-	return fmt.Sprintf("postgres://postgres:pass@localhost:%d/test?sslmode=disable", t.port)
+func (t *pgTest) dsn(dbname string) string {
```

Comment:
in Postgres, that's a schema name. the database name is "test". It's a bit different from MySQL.